service-registry/src/main/java/org/apache/servicecomb/serviceregistry/consumer/MicroserviceVersions.java

```diff
@@ -198,7 +197,6 @@ private void setInstances(List<MicroserviceInstance> pulledInstances, String rev
       List<MicroserviceInstance> inUseInstances) {
     List<MicroserviceInstance> upInstances = pulledInstances
```

Comment:
instances managed in appManager is different to old versions
maybe will cause compatible problem.
```
RegistryUtils.getServiceRegistry()
        .getAppManager()
        .getOrCreateMicroserviceVersionRule(......)
        .getVersionedCache()
```

Reply:
I created a new issue to track this : https://issues.apache.org/jira/browse/SCB-1104